### DISCUSSION TITLE
Flax CLI

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#161] [Q&A] [answered]
Chia is ...

 docker exec -it machinaris venv/bin/chia

What is the path for Flax?

**Comment by guydavis** [comment, marked as answer]
Hi! Good question. They're both on the [path in-container](https://github.com/guydavis/machinaris#container-cli) so this is the easiest approach to using CLI:
```
docker exec -it machinaris bash
chia farm summary
flax farm summary
```

To answer your question though: Flax install is at /flax-blockchain/venv/bin/flax
